jenkins/master/plugins.txt

```diff
@@ -1,6 +1,9 @@
 token-macro:2.3
 greenballs:1.15
-credentials-binding:1.12
+credentials-binding:1.13
```

Comment:
Already updated to 1.18 in #141.

Reply:
@michaelsauter - yup already fixed local :)